src/component/app.tsx

```diff
@@ -155,7 +155,7 @@ class App extends React.Component<AppProps> {
 		}
 
 		const designkitPath = PathUtils.join(appPath, 'build', 'designkit');
-		console.log(`Design kit path is: ${designkitPath}`);
+		/*console.log(`Design kit path is: ${designkitPath}`);*/
```

Comment:
It would be better to remove the `console.log`s completely

Reply:
OK.